packages/storage/test/cookies.test.ts

```diff
@@ -12,6 +12,12 @@ describe("cookieStorage", () => {
     cookieStorage.removeItem("test");
     expect(cookieStorage.getItem("test")).toBe(null);
   });
+
+  it("serializes options correctly", () => {
+    const set = vi.spyOn(document, "cookie", "set");
+    cookieStorage.setItem("test3", "good", { domain: "https://localhost:3000", path: "/", httpOnly: true, maxAge: 60 * 60 * 24 * 7 });
+    expect(set).toHaveBeenCalledWith("test3=good; domain=https://localhost:3000; path=/; http-only; max-age=604800");
```

Comment:
invalid test.  `httpOnly` is not `http-only`

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie

Reply:
Fixed.